### ISSUE TITLE
Minimap shifted

### ISSUE THREAD

**Issue by LaVADraGoN** [Bro748/World-Editor-Github#2]
I use the tool to assembly my levels, thanks for that.
But something wrong with the minimap ingame, the slugcat position is shifted a lot leftward.
I can explain what's causing this, but everything looks normal in the tool, the map looks fine in the png.
Because I let the tool generate the map, I think that's the reason. 

Maybe it has something todo with the lvl.txt, im using rwe+ to generate the levels.

I havn´t try to render the map through devtools, jet.

Here is a screenshot from ingame. 
![20230610154327_1](https://github.com/Bro748/World-Editor/assets/9208059/062d9fae-b6a4-4666-8509-384dd482efac)

Thanks for have a look into it.

**Comment by Bro748** [comment]
Hi, would you mind sending the map txt and png?

Normally an issue like this is caused by a desync between the txt and png. With the png containing the geometry map for each room, while the txt says where every room is and where to draw connections. All the room coordinates are relative to the leftmost room, so verify that the leftmost room on the map is present in the actual region and that its size matches what was loaded in the World Editor

**Comment by LaVADraGoN** [comment]
Here is a random level, i normal copy the dizzle renderer output to the modfolder for use, since is generate a level.txt together the renderer.
-Delete file

**Comment by Bro748** [comment]
Sorry, there should be a file called map_XX.txt and another called map_XX.png (with XX being the acronym for your region) and those are the files I want to see

**Comment by LaVADraGoN** [comment]
Its both together in the zipfile I attached, in the lv-rooms folder

**Comment by Bro748** [comment]
The zip contains _LV_A01.txt_ and _LV_A01_1.png_ for the actual room, and _LV_A01.txt_, _LV_A01.png_, and _LV_A01.wep_ for the source. Those are all room files, which isn't what I'm looking for.

I'm looking for files that are literally called _map_LV.txt_ and _map_LV.png_. These should be files that have been outputted by the World Editor, rather than the Level Editor, and contain all the data for the in-game minimap, hence their relevance.

**Comment by LaVADraGoN** [comment]
oh sorry i missunderstand , here is the output from the folder i generated
[LV.zip](https://github.com/Bro748/World-Editor/files/11713206/LV.zip)

**Comment by Bro748** [comment]
Right, I understand the issue now and I'll have to fix it with code. But for now, you can fix it by moving the offscreen den into the center of the region and re-rendering, as it's caused by the offscreen den being the furthest left room.

**Comment by LaVADraGoN** [comment]
ok the map seems to work now, thank you,
